android/CMakeLists.txt

```diff
@@ -35,6 +38,12 @@ string(REPLACE "-none" "" CC_FOR_TARGET ${CC_FOR_TARGET})
 # architecture than the one that comes with the ndk armv7 vs armv7a
 string(REPLACE "armv7-" "armv7a-" CC_FOR_TARGET ${CC_FOR_TARGET})
 
-add_library(${PLUGIN_NAME} SHARED "src/main/c/dummy.c")
+add_library(${PLUGIN_LIB} SHARED "src/main/c/dummy.c")
```

Comment:
See if we can avoid having a dummy C file

Reply:
Seems like we're stuck with this hack for now. The issue is that linux(and maybe other platforms) have code within that library, but for Android we didn't have such a need.